.pylintrc

```diff
@@ -0,0 +1,6 @@
+[MASTER]
```

Comment:
We don't use pylint anywhere, so we can either add it (on top of flake8), or this should go outside of git (https://stackoverflow.com/a/22449845)